src/RoadSurface/RoadSurface.tsx

```diff
@@ -0,0 +1,23 @@
+import { Line } from "react-konva";
+import React from "react";
+import { RoadSurfaceProps } from "./RoadSurface.types";
+
+const RoadSurface = (
+  { points, color = "#78736e" }: RoadSurfaceProps,
+  key: React.Key
+) => {
+  return (
+    <Line
+      key={key}
+      scaleY={-1}
+      closed
+      fill={color}
+      points={points.filter(
```

Comment:
If z-coordinates will be removed, this should be rewritten

Reply:
Done